rebar.config

```diff
@@ -19,6 +19,12 @@
 {eunit_opts, [verbose, {report,{eunit_surefire,[{dir,"."}]}}]}.
 {edoc_opts, [{preprocess, true}]}.
 
+{relx, [
+       {release, {partisan, "0.0.1"}, [partisan]},
```

Comment:
Wrong release number -- should probably be brought up to date with latest number used in most recent packaging?

Reply:
use v3.0.0